.github/ISSUE_TEMPLATE/6_backport request.yml

```diff
@@ -0,0 +1,30 @@
+name: "🔙 Backport request"
+description: Create a backport request of a bug fix.
+title: '[Backport] '
+labels: ["backport"]
+
+body:
+  - type: textarea
+    id: backports
+    attributes:
+      label: What would you like to backport?
+      description: "Provide a list of links to PRs or commit SHAs you would like to backport."
+    validations:
+      required: true
+
+  - type: textarea
+    id: graalvm_versions
+    attributes:
+      label: GraalVM versions
+      description: "List the GraalVM versions you would like to see the changes backported to."
+      value: "GraalVM for JDK 21"
+    validations:
+      required: true
+
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Describe the motivation for the backport.
+      description: "Provide a clear and concise explanation of why this backport is needed. If it fixes a reported issue please link to it."
+    validations:
+      required: true
```

Comment:
@zakkak  `\n` ? 
And an absolute nitpick: We know the rather limited set of versions to reasonably backport to, so how about `dropdown` ? https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#about-githubs-form-schema

Reply:
I didn't go for a dropdown because:

1. I expect this to be used for Oracle GraalVM backport requests as well (where the version range is not as limited as in upstream
2. I expect this to be used to request backports to branched versions as well while they are still being prepared for release (e.g. 24.2 right now), so we would need to update the dropdown every 6 months.

I will add the `\n` thanks